TreasureHunt/Assets/2.Scripts/SimpleCloudHandler.cs

```diff
@@ -99,46 +69,45 @@ public void OnStateChanged(bool scanning)
 			}
 		}
 
-		/// <summary>
 		/// Handles new search results
-		/// </summary>
 		/// <param name="targetSearchResult"></param>
 		public void OnNewSearchResult(TargetFinder.TargetSearchResult targetSearchResult)
 		{
 			// duplicate the referenced image target
 			GameObject newImageTarget = Instantiate(ImageTargetTemplate.gameObject) as GameObject;
-
 			GameObject augmentation = null;
-
 			string model_name = targetSearchResult.MetaData;
 
-
 			if (augmentation != null)
 				augmentation.transform.parent = newImageTarget.transform;
 
 			// enable the new result with the same ImageTargetBehaviour:
 			ImageTargetAbstractBehaviour imageTargetBehaviour =
 				mImageTracker.TargetFinder.EnableTracking(targetSearchResult, newImageTarget);
-
 			Debug.Log("Metadata value is " + model_name);
 			mTargetMetadata = model_name;
 
-
-
-
 			if (!mIsScanning)
 			{
 				// stop the target finder
 				mCloudRecoBehaviour.CloudRecoEnabled = true;
 			}
 		}
-
-
-		#endregion // ICloudRecoEventHandler_IMPLEMENTATION
-
-		//        void OnGUI()
-		//        {
-		//            GUI.Box(new Rect(100, 200, 200, 50), "Metadata: " + mTargetMetadata);
-		//        }
+		/*
```

Comment:
Not now but probably later for debugging because the code is not completed yet.
(I used those code for monitoring before blocked so...) Then I will add it in when it's needed!